packages/server/graphql/types/TemplateScale.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  GraphQLBoolean,
+  GraphQLID,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString
+} from 'graphql'
+import {GQLContext} from '../graphql'
+import {resolveTeam} from '../resolvers'
+import GraphQLISO8601Type from './GraphQLISO8601Type'
+import PokerTemplate from './PokerTemplate'
+import Team from './Team'
+import TemplateScaleValue from './TemplateScaleValue'
+
+const TemplateScale = new GraphQLObjectType<any, GQLContext>({
+  name: 'TemplateScale',
+  description:
+    'A team-specific template scale.',
+  fields: () => ({
+    id: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: 'shortid'
+    },
+    createdAt: {
+      type: new GraphQLNonNull(GraphQLISO8601Type)
+    },
+    isActive: {
+      type: GraphQLBoolean,
```

Comment:
nit, if you do a `resolve: ({isActive}) => !!isActive` then we can make the type NonNull.